tests/python/test_eig.py

```diff
@@ -138,7 +138,7 @@ def eigen_solve():
 
 
 @pytest.mark.parametrize("func", [_test_eig2x2_real, _test_eig2x2_complex])
-@test_utils.test(default_fp=ti.f32, fast_math=False)
+@test_utils.test(default_fp=ti.f32, fast_math=False, dynamic_index=True)
```

Comment:
```suggestion
@test_utils.test(default_fp=ti.f32, fast_math=False)
```
This shouldn't be changed.